extensions/amp-consent/0.1/consent-policy-manager.js

```diff
@@ -76,14 +90,22 @@ export class ConsentPolicyManager {
     return isExperimentOn(win, 'multi-consent');
   }
 
+  /**
+   *
+   * @param {string} consentInstanceId
+   */
+  setLegacyConsentInstanceId(consentInstanceId) {
+    this.consentInstanceIdDepr_ = consentInstanceId;
+    this.init_();
+  }
+
   /**
    * Register the policy instance
    * Example policy config format:
    * {
    *   "waitFor": {
    *     "consentABC": [], // Can't support array now.
```

Comment:
Since we now only have one consent, does this format still make sense? 🤔

Reply:
Replied here https://github.com/ampproject/amphtml/pull/20089#discussion_r244237863